### PR TITLE
Add tests for `uv cache clean`

### PR DESCRIPTION
**Pull request by charliermarsh**

### CODE REVIEW

File: crates/uv/tests/cache_clean.rs [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+
+mod common;
+
+/// Create a `cache clean` command with options shared across scenarios.
+fn clean_command(context: &TestContext) -> Command {
+    let mut command = Command::new(get_bin());
+    command.arg("cache").arg("clean");
+    context.add_shared_args(&mut command);
+    command
+}
```
Comment (zanieb) [inline crates/uv/tests/cache_clean.rs:21]: Technically this is bad form, can you add it to the `TestContext` to match the other commands?

Comment (charliermarsh) [reply crates/uv/tests/cache_clean.rs:21]: Yup.

Comment (charliermarsh) [reply crates/uv/tests/cache_clean.rs:21]: (I cribbed this from `cache_prune.rs` so changing that too.)

Comment (zanieb) [reply crates/uv/tests/cache_clean.rs]: Thanks!

**Review by zanieb — APPROVED** [review]

**Pull request merged**